demos/python_demos/smart_library_demo/smart_library_demo.py

```diff
@@ -0,0 +1,252 @@
+import sys, os
+import argparse
+import numpy as np
+import cv2 as cv
+import json
+
+sys.path.insert(0,'src/modules')
+import face_recognizer as fr 
+import book_recognizer as br
+
+sys.path.insert(0, 'src/infrastructure')
```

Comment:
This solutions works only if the working directory is `open_model_zoo/demos/python_demos/smart_library_demo`. It is possible to import modules like `from src.modules import face_recognizer as fr`.